### ISSUE TITLE
Few pointers for others

### ISSUE THREAD

**Issue by motoridersd** [bsmithio/OPNsense-Dashboard#34]
First, I want to thank you for this work. This is the first time I've gotten a dashboard to work because I was always trying to do this on my own without containers. This was so much easier.

A few things I got stuck on that might help others:

- Portainer runs on the same port as graylog, so if you are using this in your container/box, you will need to re-configure the port it uses. Pay attention to the output of docker compose, this is where I saw Graylog failing to bind to port 9000
- The MaxMind API token takes a few minutes to activate, so wait a bit before you run the commands
- Enable all Telegraf monitoring items, in case they are disabled. I had them disabled for some reason
- I missed the "reorder" part of the Graylog configuration. So if you are not getting results in your map, make sure your Message Processors Configuration matches the screenshot

**Comment by PaulePils** [comment]
Regarding your point for Portainer:
I was curious why I cant get the Firewall-data. But when I access Portainer, I use the port 9443 and not 9000. After looking it up there is a legacy port 9000 for Portainer. Is there a way to change/deactive it?

**Comment by motoridersd** [comment]
> Regarding your point for Portainer: I was curious why I cant get the Firewall-data. But when I access Portainer, I use the port 9443 and not 9000. After looking it up there is a legacy port 9000 for Portainer. Is there a way to change/deactive it?

Yes, you can change the port used by Portainer. I forget how I did it

**Comment by PaulePils** [comment]
I got it running. I deleted the LXC and started from scratch but this time without Portainer.
After a few bugs with graylog (there was the error in preflight check) I got it finally working :-)

**Comment by ripple7511** [comment]
@PaulePils , I also am running Grafana in an LXC. How did you do the MindMax portion? I'm not sure where to drop the various files when not using docker.

**Comment by PaulePils** [comment]
@ripple7511, I installed Docker and Compose to use the template and did not install it directly in the LXC
